concord-server/server/src/main/java/com/rabidgremlin/concord/resources/PhrasesResource.java

```diff
@@ -175,6 +182,12 @@ public Response voteForPhrase(@ApiParam(hidden = true) @Auth Caller caller, @Pat
 
     votesDao.upsert(phraseId, label.getLabel(), caller.getToken());
 
+    // are we marking trashed labels as completed?
+    if (completeOnTrash && StringUtils.equals(LABEL_TRASH, label.getLabel()))
+    {
+      phrasesDao.markPhrasesComplete(Arrays.asList(phraseId), Arrays.asList(label.getLabel()));
```

Comment:
Could use LABEL_TRASH instead of label.getLabel() here to save a method call

Reply:
It's not public... however I have plans to do some refactoring to pull all the various hard coded strings into a utils/globals/consts class.